test/http_to_do_test.exs

```diff
@@ -39,4 +47,74 @@ defmodule HTTPServerTest.ToDo do
 
     assert API.create(handle_resp) == error_message
   end
+
+  test "Can delete \"{\"todo1\":\"Act\"}\" from data file" do
+    todo_to_be_deleted = "1"
+
+    API.delete(todo_to_be_deleted)
+
+    file_contents =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    refute file_contents[todo_to_be_deleted]
+  end
+
+  test "Can delete \"{\"todo4\":\"Teardown\"}\" from data file" do
+    todo_to_be_deleted = "4"
+
+    API.delete(todo_to_be_deleted)
+
+    file_contents =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    refute file_contents[todo_to_be_deleted]
+  end
+
+  test "The file will not change when id = 5" do
+    todo_to_be_deleted = "5"
+
+    data_before_delete =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    API.delete(todo_to_be_deleted)
+
+    data_after_delete =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    assert data_before_delete == data_after_delete
+  end
```

Comment:
this test names is really specific to the test implementation.  What is actually being tested here? and can the name reflect that intent?  its easy to accomplish with good names.
```suggestion
  test "Cannot delete a ToDo with an id that doesn't exist" do
    non_existant_todo_id = "5"

    test_todos =
      File.read!(@file_path)
      |> JSON.decode!()

    API.delete(non_existant_todo_id)

    test_todos_after_delete =
      File.read!(@file_path)
      |> JSON.decode!()

    assert test_todos == test_todos_after_delete
  end
```

Reply:
This same naming is persistent through most of the tests in this file.

Reply:
I guess what really stands out to me is, the test reads like if the id of a todo is ever 5, then this application will not delete it.  "The fill will no change when the id = 5"